vllm/entrypoints/openai/rpc/client.py

```diff
@@ -32,6 +33,10 @@
 INPROC_PROXY_PATH = f"inproc://{uuid4()}"
 
 
+class ClientClosedError(Exception):
```

Comment:
Can we call this `RPCClientClosedError`? It should make it clearer when used in the `api_server.py`

Reply:
Yeah sure!